charts/intel-qat-resource-driver/values.yaml

```diff
@@ -19,14 +19,22 @@ serviceAccount:
 
 kubeletPlugin:
   podAnnotations: {}
-  tolerations:
-    - key: node-role.kubernetes.io/master
-      operator: Exists
-      effect: NoSchedule
-    - key: node-role.kubernetes.io/control-plane
-      operator: Exists
-      effect: NoSchedule
   nodeSelector:
-    {}
+    feature.node.kubernetes.io/qat: "true"
     #node-role.kubernetes.io/control-plane: ""
+  tolerations:
+    # - key: node-role.kubernetes.io/master
+    #   operator: Exists
+    #   effect: NoSchedule
+    # - key: node-role.kubernetes.io/control-plane
+    #   operator: Exists
+    #   effect: NoSchedule
+    - key: "node.kubernetes.io/qat"
+      operator: "Exists"
+      effect: "NoSchedule"
```

Comment:
Just a question: what this toleration does? The NFD label seems to be `feature.node.kubernetes.io/qat`, is there a health management software that puts this taint when QAT goes unhealthy or when is this taint applied?

Reply:
NFD has support for tainting the node with a device specific taint: https://nfd.sigs.k8s.io/usage/customization-guide#node-tainting

Adding toleration for it, even if it would not be used yet, is fine I think, as taniting accelerator nodes to avoid unrelated workloads being scheduled on them, is common cluster practice.

Reply:
I think this toleration should be only added if NFD is configured with the "enable-taints" option? Or this toleration should have a comment and a link to the NFD documentation.